ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -122,7 +135,24 @@
   [context task-id]
   (get-bulk-update-task-status (context->db context) task-id))
 
-(defn-timed create-bulk-update-status
-  "Saves the map of provider id acl hash values"
+(defn-timed get-bulk-update-collection-statuses-for-task
+  [context task-id]
+  (get-bulk-update-task-collection-status (context->db context) task-id))
+
+(defn-timed create-bulk-update-task
+  "Creates all the rows for bulk update status tables - task status and collection
+  status. Returns task id"
   [context provider-id json-body concept-ids]
   (create-and-save-bulk-update-status (context->db context) provider-id json-body concept-ids))
+
+(defn-timed update-bulk-update-task-collection-status
+  "For the task and concept id, update the collection to the given status with the
+  given status message"
+  [context task-id concept-id status status-message]
+  (def context context)
+  (update-bulk-update-collection-status (context->db context) task-id concept-id
+    status status-message))
+
+(comment
```

Comment:
I am leaving it for now because it's immensely helpful for me to have it especially since this is still very much in development.